ui/packages/ui/src/app/components/formHelpers/FormMaskHashSaltComponent.tsx

```diff
@@ -86,21 +90,52 @@ export const FormMaskHashSaltComponent: React.FunctionComponent<IFormMaskHashSal
     return "";
   };
 
+  const onToggle = (open: boolean) => {
+    setOpen(open)
+  };
+
+  const clearSelection = () => {
+    setSelected(false)
+    setOpen(false)
+  };  
+
+  const onSelect = (e, selection: boolean, isPlaceholder: boolean) => {
+    if (isPlaceholder) {
+      clearSelection();
+    }
+    else {
+      setSelected(selection)
+      setOpen(false)
+      let newVal = getColsValue(field.value)+"&&"+selection+"||"+getSaltValue(field.value);
```

Comment:
Local build is giving an error - need to change to 'const newVal = ...'

Reply:
fixed.